server/routes/index.ts

```diff
@@ -745,21 +747,20 @@ export function defineRoutes(router: IRouter) {
   router.delete(
     {
       path: `${API_PREFIX}/configuration/cache`,
-      validate: false,
+      validate: {
+        body: schema.object({
```

Comment:
Should dataSourceId(s) always be passed in as query params?

Reply:
Yea, since we control all the places this is called (only in the get started tab) I think it's fine to require it. This isn't using query params though it's only an internal implementation. However I think this is a two way door decision and once more routes are modified we can easily refactor it to a cleaner way if we find one

Reply:
If that makes sense can you merge? Thanks!